packages/dds/tree/src/shared-tree-core/messageCodecs.ts

```diff
@@ -52,7 +52,7 @@ export function makeMessageCodec<TChangeset>(
 }
 
 /**
- * @privateRemarks - Exported for testing.
+ * @privateRemarks Exported for testing.
```

Comment:
Didn't add a summary to this, since I don't know what it is. But the `@privateRemarks` syntax was wrong, so I fixed that.